src/Common/src/Interop/Windows/kernel32/Interop.GetFileAttributesEx.cs

```diff
@@ -68,10 +67,8 @@ internal unsafe struct WIN32_FIND_DATA
             internal uint nFileSizeLow;
             internal uint dwReserved0;
             internal uint dwReserved1;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 260)]
-            internal string cFileName;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 14)]
-            internal string cAlternateFileName;
+            internal FixedString.Size260 cFileName;
+            internal FixedString.Size14 cAlternateFileName;
```

Comment:
I would use:

```
internal fixed char cFileName[260];
internal fixed char cAlternateFileName[14];
```

I do not think that the FixedString helper type proposed here is an improvement.

> We could stick to raw char[] and wait for help from the compiler 

I do not see how this can ever help. The point is to make this struct blittable to eliminate marshalling overhead. `char[]` field is always going to require marshalling.